pkgs/servers/sql/postgresql/ext/tsja.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, fetchurl
+, nixosTests
+, stdenv
+
+, mecab
+, postgresql
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tsja";
+  version = "0.5.0";
+
+  src = fetchurl {
+    url = "https://www.amris.jp/tsja/tsja-${version}.tar.xz";
+    hash = "sha256-Z8iwMQwS5mjnx8D8hgHB/5rxrTehrXhBRFcSUJzUAGI=";
+  };
+
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace /usr/local/pgsql ${postgresql} \
+      --replace -L/usr/local/lib "" \
+      --replace -I/usr/local/include ""
+    substituteInPlace tsja.c --replace /usr/local/lib/mecab ${mecab}/lib/mecab
+  '';
+
+  buildInputs = [ mecab postgresql ];
+
+  installPhase = ''
+    mkdir -p $out/lib $out/share/postgresql/extension
+    mv libtsja.so $out/lib
+    mv dbinit_libtsja.txt $out/share/postgresql/extension/libtsja_dbinit.sql
+  '';
+
+  passthru.tests.tsja = nixosTests.tsja;
+
+  meta = with lib; {
+    description = "PostgreSQL extension implementing Japanese text search";
+    homepage = "https://www.amris.jp/tsja/index.html";
+    maintainers = with maintainers; [ chayleaf ];
+    platforms = postgresql.meta.platforms;
+    license = licenses.postgresql;
```

Comment:
Not that acquainted with postgres packages, but the source tarball notes in `docs/COPYING`:

    Feml is copyrighted free software by KOYAMA Hiro <tac@amris.co.jp>.
    You can redistribute it and/or modify it under the terms of the GPL version 2 (see the file GPL).

Reply:
thanks for catching this!